pkgs/development/python-modules/uncertainties/default.nix

```diff
@@ -22,7 +22,8 @@ buildPythonPackage rec {
   src = fetchFromGitHub {
     owner = "lmfit";
     repo = "uncertainties";
-    tag = version;
+    # tag = version; # will fail
+    rev = "1238d37c6ed87d018e696589716de488e7d9eca7";
```

Comment:
In what way did `tag = version;` fail? It worked on my machine when I tried just now.

Reply:
I just checked both tag and revision and both worked. I would drop this commit since it doesn't really make sense.

That README comment isnt about tags vs commits anyway.

Reply:
Re-checked this myself, and you are correct. So, done.

> README

It is about short revisions, but tag are also, essentially, short revisions, since GitHub did return 404 for me for this, I think it applies.